modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommercePaymentTerms.testcase

```diff
@@ -0,0 +1,57 @@
+@component-name = "portal-commerce"
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Order Management";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		Navigator.openURL();
+
+		if (IsElementPresent(locator1 = "UserBar#USER_AVATAR_IMAGE")) {
+			User.logoutPG();
+		}
+
+		User.firstLoginPG();
+
+		CommerceConfiguration.commerceAdminTearDown();
+
+		CommerceConfiguration.commerceInitializersTearDown();
+	}
+
+	@description = "Delete a new payment term to Terms and Conditions."
+	@priority = "3"
+	test DeleteAPaymentTerm {
```

Comment:
Please change the test name to reflect the expected behavior. "CanDeleteAPaymentTerm" would be good since it describes what someone analyzing the test should expect.